lib/pages/chat/chat_pinned_events/pinned_events_view.dart

```diff
@@ -218,7 +218,7 @@ class _PinnedEventsContentWidget extends StatelessWidget {
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
             Text(
-              L10n.of(context)!.countPinnedMessage(countPinnedEvents),
+              L10n.of(context)!.countPinnedMessage(countPinnedEvents + 1),
```

Comment:
you should defined `pinnedEventsOrder = countPinnedEvents + 1`

Reply:
should not have number if it is the first pin

Reply:
done
https://github.com/linagora/twake-on-matrix/assets/48354990/c41f0fd6-c472-41ec-a2e4-9e112d270b5a